ntbs-service-unit-tests/Services/UserSearchServiceTest.cs

```diff
@@ -177,6 +177,26 @@ public async Task OrderAndPaginateQueryableAsync_ReturnsFamilyNameMatchingUsers(
             Assert.Equal(expectedResults, results.users);
         }
 
+        [Fact]
+        public async Task OrderAndPaginateQueryableAsync_DoesNotReturnInactiveFamilyNameMatchingUsers()
+        {
+            // Arrange
+            const string searchString = "UserT";
+            var expectedResults = new List<User> { DefaultFamilyNameCaseManagers[2] };
+            var usersToReturn = DefaultFamilyNameCaseManagers;
+            usersToReturn[1].IsActive = false;
```

Comment:
This is a bit of an anti-pattern. You've mutated the `DefaultFamilyNameCaseManagers` object which is static, and therefore shared across multiple tests. This means that for any tests that run after this one, the second item in the list will not be active. In general, your tests ought to be completely identical, regardless of the order in which they are run.

There are many possible solutions here, a few of which are:
- Make `DefaultFamilyNameCaseManagers` a class member so that it gets created on every test run (XUnit creates a new instance of the class for every test)
- Add an extra user to `DefaultFamilyNameCaseManagers` which is not active (and, if necessary, adjust the other tests to account for this) 
- Use different seed data entirely for this test

Reply:
I've gone with option 3, which matches what I've done in the rest of the suite and also the other test I added!